spyder/plugins/editor/widgets/codeeditor/lsp_mixin.py

```diff
@@ -117,6 +117,8 @@ class LSPMixin:
     #: Signal emitted when processing code analysis warnings is finished
     sig_process_code_analysis = Signal()
 
+    sig_code_folding_info = Signal(tuple)
```

Comment:
```suggestion
    #: Signal emitted to tell cloned editors that they need to update their
    # code folding info
    sig_update_code_folding_info = Signal(tuple)
```

* Add comment to describe what the signal does.
* Use a more meaningful name for it.